flake.nix

```diff
@@ -27,7 +27,9 @@
         # The usual flake attributes can be defined here, including system-
         # agnostic ones like nixosModule and system-enumerating ones, although
         # those are more easily expressed in perSystem.
-
+        defaultPackage.x86_64-linux = self.packages.x86_64-linux.default;
+        defaultPackage.aarch64-linux = self.packages.aarch64-linux.default;
+        defaultPackage.aarch64-darwin = self.packages.aarch64-darwin.default;
```

Comment:
The `defaultPackage` attribute isn't needed anymore since many many releases of flakes. `nix` will look at `packages.<system>.default` anyway, so these lines can be removed without any change in behavior.

```suggestion
```